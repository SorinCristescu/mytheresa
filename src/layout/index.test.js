import React from 'react';
import { shallow } from 'enzyme';
import Layout from './index';

it('to render Layout component', () => {
  expect(shallow(<Layout />)).toMatchSnapshot();
});
