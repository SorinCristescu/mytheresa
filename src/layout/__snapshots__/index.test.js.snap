// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to render Layout component 1`] = `
<div>
  <Header />
  <styled.div />
  <Footer />
</div>
`;
